### PR TITLE
Return an array of avatar urls in different sizes for Comments and Users

### PR DESCRIPTION
**Pull request by rachelbaker**
- Add rest_get_avatars and deprecate rest_get_avatar
- Update Comment controller to use rest_get_avatars 
- Update Users controller to use rest_get_avatars

### CODE REVIEW

**Comment by rachelbaker** [comment]
Fixes #1330 and #1290

We now return an array of avatar urls that are keyed on the image size.

Example:

```
  "avatar_urls": {
    "24": "http://0.gravatar.com/avatar/06e92fdf4a9a63441dff65945114b47f?s=24&d=mm&r=g",
    "48": "http://0.gravatar.com/avatar/06e92fdf4a9a63441dff65945114b47f?s=48&d=mm&r=g",
    "96": "http://0.gravatar.com/avatar/06e92fdf4a9a63441dff65945114b47f?s=96&d=mm&r=g"
  },
```

File: lib/endpoints/class-wp-rest-comments-controller.php [hunk 1]
```diff
@@ -700,9 +700,9 @@ public function get_item_schema() {
 					'type'         => 'integer',
 					'context'      => array( 'view', 'edit', 'embed' ),
 					),
-				'author_avatar_url' => array(
-					'description'   => 'Avatar URL for the object author.',
-					'type'          => 'string',
+				'author_avatar_urls' => array(
+					'description'   => 'Avatar URLs for the object author.',
+					'type'          => 'array',
 					'format'        => 'uri',
```
Comment (rachelbaker) [inline lib/endpoints/class-wp-rest-comments-controller.php]: @danielbachhuber can I still use the `uri` format with the `array` type?

**Comment by rachelbaker** [comment]
@WP-API/amigos #reviewmerge

File: lib/endpoints/class-wp-rest-comments-controller.php (hunk 1 above)
Comment (danielbachhuber) [inline lib/endpoints/class-wp-rest-comments-controller.php]: You're actually returning an object now, so it would be an object with specific properties

Comment (rachelbaker) [inline lib/endpoints/class-wp-rest-comments-controller.php]: http://media.giphy.com/media/8WdsK61D9YOOc/giphy.gif

**Comment by rachelbaker** [comment]
Hold off on a merge, I need to fix the schema.

**Comment by rachelbaker** [comment]
@WP-API/amigos #reviewmerge

**Comment by rmccue** [comment]
:+1:

**Pull request merged**